src/engraving/dom/mscore.h

```diff
@@ -77,6 +76,8 @@ static constexpr double DPMM = DPI / INCH;
 // and back-compatibility, but this will be multiplied x2 during layout.
 static constexpr double MUSICAL_SYMBOLS_DEFAULT_FONT_SIZE = 10.0;
 
+static constexpr double UI_ICONS_DEFAULT_FONT_SIZE = 12.0;
```

Comment:
Why did this need to go from 16 to 12?

Reply:
Yes, that was the next step I'm planning. The disadvantage is that constants.h will become such a weird header that's a collection of random things, and changing one of them will cause the entire code base to be recompiled. I'll see if I can do something nice about that.